docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -448,7 +448,9 @@ subquestion: |
   * ${ al_opt }
   % endfor
 
+  % if not get_config("verbose error messages") == False:
```

Comment:
Should this be under `assembly line`? Not sure what we've been doing there, I do know there's an open issue for it.

Reply:
No, this is an existing Docassemble configuration option that I figured we should respect. It defaults to `True` but it's not defined in the normal config which is why we're doing an explicit check for `False`. https://docassemble.org/docs/config.html#verbose%20error%20messages

Reply:
Gotcha, that's fair.